apps/judicial-system/web/messages/Core/tables.ts

```diff
@@ -103,4 +103,9 @@ export const tables = defineMessages({
     description:
       'Notaður sem titill fyrir birtingarstaða dálk í lista yfir mál.',
   },
+  postponed: {
```

Comment:
The same string is in `apps/judicial-system/web/src/routes/Shared/Cases/ActiveCases.strings.ts`. Please delete that file.

Reply:
Actually, thist string is still being used in `ActiveCases.tsx`. That file defines the `ActiveCases` component which is now only used for prosecution users. The `ActiveCases` component will be removed together with the `string` file once we rewrite it using the `Table` component. So I will leave it for now.